jac-uk/admin
# #2428 refresh live exercises

## What's included?

- Remove the "This is a new service..." line
- Capitalise initial 'e' in page title, i.e. Live Exercises
- Amend column heading to single word titles, i.e. Reference, Name, Open, Close, Status, Applications
- Make the default sort by Reference Descending, i.e. lowest number first
- Have the entire row be the link, i.e. no underlined hyperlink, click anywhere in the row to go to the relevant exercise, grey highlight when rolled over
- Sign out link at top right becomes Display name, i.e. Addy, Nick
- Sign out link has a Downward arrow symbol to the left (to signify submenu)
- When Addy, Nick is clicked, a sub menu appears containing the options Sign out, My favourites - could be on a grey bar that stretches across the entire screen (this could either 'float' on top of text or push text down the screen when open
- Exercises link in top navigation has a Downward arrow symbol to the left (to signify submenu)
- When Exercises is clicked, a sub menu appears containing the items: Live Exercises, Create exercise, View archived exercises (in that order)

Closes #2428 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the following page (and login if necessary): https://jac-admin-develop--pr2465-feature-2428-refresh-hxenxrk4.web.app/
- Ensure the "This is a new service..." line isnt displayed
- Ensure the title says 'Live Exercises'
- Ensure column headings are: Reference, Name, Open, Close, Status, Applications
- Ensure the data initially sorts by Reference Descending, i.e. lowest number first
- Ensure the entire row is a link, i.e. no underlined hyperlink, click anywhere in the row to go to the relevant exercise
- Ensure when you hover over a row it highlights the entire row in grey
- Ensure the Sign out link at top right becomes Display name, i.e. Addy, Nick
- Ensure the Sign out link has a Downward arrow symbol to the left (to signify submenu)
- When the Display Name is clicked, a sub menu appears containing the options Sign out, My favourites
- Ensure the Exercises link in top navigation has a Downward arrow symbol to the left (to signify submenu)
- When Exercises is clicked, a sub menu appears containing the items: Live Exercises, Create exercise, View archived exercises (in that order)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
